src/System.Diagnostics.Process/tests/ProcessTests.netstandard1.7.cs

```diff
@@ -109,5 +110,74 @@ public void TestWorkingSet()
 
             Assert.True(_process.WorkingSet > 0);
         }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void Process_StartInvalidNamesTest()
+        {
+            Assert.Throws<InvalidOperationException>(() => Process.Start(null, "userName", new SecureString(), "thisDomain"));
+            Assert.Throws<InvalidOperationException>(() => Process.Start(string.Empty, "userName", new SecureString(), "thisDomain"));
+            Assert.Throws<Win32Exception>(() => Process.Start("exe", "userName", new SecureString(), "thisDomain"));
+        }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void Process_StartWithInvalidUserNamePassword()
+        {
+            SecureString password = AsSecureString("Value");
+
+            Assert.Throws<Win32Exception>(() => Process.Start("exe", "userName", password, "thisDomain"));
+        }
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void Process_StartTest()
+        {
+            string currentProcessName = $"{Process.GetCurrentProcess().ProcessName}.exe";
+            string userName = string.Empty;
```

Comment:
So this is currently testing with an empty username so that it doesn't create the process with logon so that we can actually test that the `Process` is currently being created correctly such as `ProcessStartInfo`

Will add the current user garbage password test cause I already had the garbage password test :).

Thanks